docs/source/index.rst

```diff
@@ -36,6 +36,36 @@ user:
     print(kennethreitz.followers_count)
 
     followers = [str(f) for f in gh.followers('kennethreitz')]
+    
+    # create a repository
+    gh.create_repository(name='Repo_Name')
+    
+    # grab the repo
+    repos = list(gh.repositories())
```

Comment:
If we're retrieving the repository directly, do we need to list repositories and is the "grab the repo" comment still valid?

Reply:
It's up to you!
I basically wanted to convey 2 things:
1. How to create an object (repo, issue, milestone)
2. Assuming many of these are created, how to grab a specific object from a list of objects.

It's tough here because yes, we just created the object so it would be a lot easier to just have it be returned.

For the sake of documentation, would it be more beneficial for a new user to just see how one can create the objects, or interact with objects already existing in a repository? Thoughts?